src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -302,6 +311,21 @@ public class OpenIdConnectAuthenticationHandler : AuthenticationHandler<OpenIdCo
                     _configuration = await Options.ConfigurationManager.GetConfigurationAsync(Context.RequestAborted);
                 }
 
+                // Redeeming authorization code for tokens
+                if (string.IsNullOrEmpty(message.IdToken) && !string.IsNullOrEmpty(message.Code))
```

Comment:
That is in our POR to use code in id_token + code response type too. Currenlty, we are redeeming it only for the sigin flow ("Code" response type)